README.md

```diff
@@ -34,7 +45,7 @@ the end of your password as it will break the login.
 
 # Execution
 
-Simply run `python downloader.py` and watch it fetch your documents.
+Simply run `python3 downloader.py` and watch it fetch your documents.
```

Comment:
I understand that you might want to be explicit in the makefile but in documentation I don't see the need to mention an explicit version - especially since python2 is officially unsupported over 3 years now

Reply:
Yes, correct. But on multiple Linux distributions that means that "python" will either not exist or still be the outdated not anymore upstream supported python 2.

Reply:
excluded that change from this PR for now